### PR TITLE
Fix Broken MaxText CI

### PR DESCRIPTION
**Pull request by dipannita08**
# Description

MaxText CI broken due to restructuring regression. This fixes the bad imports.

FIXES: b/486187973

# Tests

Github CI

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code. For an optional AI review, add the `gemini-review` label.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run end-to-end tests tests and provided workload links above if applicable.
- [x] I have made or will make corresponding changes to the doc if needed, including adding new documentation pages to the relevant Table of Contents (toctree directive) as explained in [our documentation](https://maxtext.readthedocs.io/en/latest/development.html#adding-new-documentation-files).

### CODE REVIEW

File: tests/unit/pipeline_parallelism_test.py [hunk 1]
```diff
@@ -26,13 +26,16 @@
 import jax.numpy as jnp
 from jax.sharding import Mesh
 from MaxText import pyconfig
-from maxtext.common.gcloud_stub import is_decoupled
 from MaxText.common_types import MODEL_MODE_TRAIN
 from MaxText.globals import MAXTEXT_ASSETS_ROOT
+from maxtext.common.gcloud_stub import is_decoupled
```
Comment (A9isha) [inline tests/unit/pipeline_parallelism_test.py:31]: is this switch just for lint issue?

File: tests/unit/pipeline_parallelism_test.py [hunk 2]
```diff
@@ -26,13 +26,16 @@
...
 from maxtext.layers import nnx_wrappers
 from maxtext.layers import pipeline
-from maxtext.layers import simple_layer
+from maxtext.models import deepseek
+from maxtext.models import simple_layer
```
Comment (A9isha) [inline tests/unit/pipeline_parallelism_test.py:35]: do these two have to be in separate lines?

**Review by A9isha — APPROVED** [review]
Thank you Dipannita!

File: tests/unit/pipeline_parallelism_test.py (hunk 1 above)
Comment (dipannita08) [reply tests/unit/pipeline_parallelism_test.py:31]: Correct!

File: tests/unit/pipeline_parallelism_test.py (hunk 2 above)
Comment (dipannita08) [reply tests/unit/pipeline_parallelism_test.py:35]: Just just reverted to the state prior to the regression patch to fix the tests. We can do additional forms of cleanup as follow up PRs.
